build.gradle.kts

```diff
@@ -88,7 +88,8 @@ sonar {
             "dataland-frontend/src/components/forms/parts/elements/derived/NaceCodeTree.ts," +
                 "dataland-frontend/src/components/resources/frameworkDataSearch/lksg/LksgDataModel.ts," +
                 "dataland-backend/src/main/kotlin/db/migration/V1_1__CreateBackendTables.kt," +
-                "dataland-frontend/src/components/resources/frameworkDataSearch/p2p/P2pDataModel.ts",
+                "dataland-frontend/src/components/resources/frameworkDataSearch/p2p/P2pDataModel.ts," +
+                "dataland-frontend/src/components/resources/frameworkDataSearch/sme/SmeDataModel.ts",
```

Comment:
Is the comma at the end actually necessary? Looks confusing

Reply:
Yes, this is how the linter desires it since the last manual maintenance. I'm actually happy with it because it is now in line with how the kotlin linter behaves in this regard.